gitbook-docs/market/overview.md

```diff
@@ -1,63 +1,81 @@
+to buy and sell future values of Bitcoin via perpetuates. These are called derivative markets,
+as you do not own the underlying instrument but are just speculating on its price.
 
-\=nil;'s proof market is more on the lines of a Spot Market rather than a speculative value of proof generation.
+`\=nil;` Proof Market is more on the lines of a Spot Market rather than a speculative
+value of proof generation.
 
 {% hint style="info" %}
-The beta of the proof market is centralised.
+The beta version of the Proof Market is centralized.
 {% endhint %}
 
+## Instruments/circuits
 
-
-## Instruments/Circuits
-
-As mentioned above, markets tend to arrive at a fair value for an instrument through interactions between buyers and sellers. In =nil's proof markets, the instruments are Circuits (initially generated by the zkLLVM). These circuits are serialised byte codes created by the proof system, which was compiled with the zkLLVM compiler.&#x20;
+Markets tend to arrive at a fair value for an instrument through interactions between buyers and sellers.
+In `=nil;` Proof Market, the instruments are circuits — serialized byte codes created by the proof system,
+compiled with the zkLLVM compiler.
 
 {% hint style="info" %}
-The instruments will require approval by =nil;Foundation to trade on Proof Market.
+To be traded on the Proof Market, the instruments will require approval by `=nil;` Foundation.
 {% endhint %}
 
-
-
 ## Entities
 
-The marketplace consists of the following entities.
-
-* **Proof Requester**: This can be an application like a bridge requesting balance or a user interested in the cross-cluster operation and/or trust-less data access.
-* **Proof Producer**: This is an entity that will generate the proofs for the requests/orders made by the Proof Requester.
+The marketplace consists of the following entities:
 
-## Orders Types
+* **Proof requester** — an application like a bridge requesting balance or a user
+    interested in a cross-cluster operation or trustless data access.
+* **Proof producer** — an entity that will generate the proofs for the requests/orders
+    made by the proof requester.
 
-* **Buy Order**: A buy order is posted by a proof requester. Each buy order comprises of following parameters:
-  * **Circuit**: The instruments for which this order is placed. ex: MINA state-proof circuit or Solana state-proof circuit.
-  * **Cost**: Amount the user is willing to pay for the proof. This is a number represented in USD.
-  * **Order Time Out**: Once the order enters the marketplace, this defines how long it should stay in the order book. If it times out before being matched, it is purged. This is a number expressed in minutes.
-  *   **Generation Time**:  Once the order is matched, this specifies the maximum time the proof generator has to publish the proof. This is an optional parameter. This is a number expressed in minutes.&#x20;
+## Orders types
 
+* **Buy order**: A buy order is posted by a proof requester.
```

Comment:
Seen it already. Did the previous author just copy-paste this part?

Reply:
Probably. What should we do, describe it here and link from the Web Interface page or vice versa?

Reply:
Please feel free to tag me for any clarifications. Which page is this duplicated from?

Reply:
@hgedia that's not exactly duplication, but the descriptions are similar in most parts
https://docs.nil.foundation/proof-market/market/overview#orders-types
https://docs.nil.foundation/proof-market/market/front-end#create-orders

Maybe we should only describe order types on the Terminology page, link it from the Frontend, and point that circuit presented as a statement should be put into Public inputs?

Reply:
> Maybe we should only describe order types on the Terminology page, link it from the Frontend

Done that